troubleshooting/faq.md

```diff
@@ -161,6 +162,18 @@ So it’s important to understand the different storage driver options available
 
 We recommend that you use the `overlay` driver, rather than `devicemapper`; for help configuring this [please see the following tutorial](https://docs.docker.com/engine/userguide/storagedriver/overlayfs-driver/#configure-docker-with-the-overlayoverlay2-storage-driver)
 
+## How do I upgrade npm Enterprise license?
```

Comment:
Let's change this to:

"How do I update my npm Enterprise license?"

And switch `upgrade` to update elsewhere.